charts/vmclarity/templates/apiserver/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-apiserver" (include "vmclarity.names.fullname" .) }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
`| quote` is not needed

Reply:
a number is a valid namespace name, for example 0123, unless namespace is quoted the kubernetes request fails because yaml converts it to an int instead of a string.